src/content/docs/apm/agents/ruby-agent/configuration/distributed-tracing-ruby-agent.mdx

```diff
@@ -100,26 +94,32 @@ For more help finding your traces in the UI:
 
 * [Understand and use the distributed tracing UI](/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui)
 * [Query distributed trace data](/docs/understand-dependencies/distributed-tracing/ui-data/query-distributed-trace-data)
+  </Step>
+</Steps>
 
-### Set up Infinite Tracing [#infinite-tracing]
+## Infinite Tracing [#infinite-tracing]
 
 Standard distributed tracing for APM agents ([above](#quick-start-apm)) captures up to 10% of your traces, but if you want us to analyze all your data and find the most relevant traces, you can set up Infinite Tracing.
```

Comment:
I'd remove the `above` link and added to the `Standard distributed tracing for APM agents` text.

Reply:
@nbaenam I fixed this!